afanasy/src/project.cmake/watch/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ else()
 	"${CMAKE_CURRENT_SOURCE_DIR}/../../watch/wndcustomdata.h"
 	"${CMAKE_CURRENT_SOURCE_DIR}/../../watch/wndcustomizegui.h"
 	"${CMAKE_CURRENT_SOURCE_DIR}/../../watch/wndtask.h"
+	"${CMAKE_CURRENT_SOURCE_DIR}/../../watch/item.h"
+	"${CMAKE_CURRENT_SOURCE_DIR}/../../watch/itemjobtask.h"
 	)
```

Comment:
I updated this `project.*` directory, because it is the only one I use. It should be updated in other build systems as well I guess.